src/cli_util.rs

```diff
@@ -1234,6 +1234,46 @@ pub fn write_commit_summary(
     Ok(())
 }
 
+pub fn write_config_entry(
+    ui: &mut Ui,
+    path: &str,
+    value: config::Value,
+) -> Result<(), CommandError> {
+    match value.kind {
+        config::ValueKind::Table(table) => {
+            for (key, table_val) in table {
+                let key_path = match path {
+                    "" => key,
+                    _ => format!("{path}.{key}"),
+                };
+                write_config_entry(ui, key_path.as_str(), table_val)?;
+            }
+        }
+        _ => writeln!(ui, "{path}={}", serialize_config_value(value))?,
+    };
+    Ok(())
+}
+
+fn serialize_config_value(value: config::Value) -> String {
+    match value.kind {
+        config::ValueKind::Table(table) => format!(
+            "{{{}}}",
+            table
+                .into_iter()
+                .map(|(k, v)| format!("{k}={}", serialize_config_value(v)))
+                .join(", ")
+        ),
```

Comment:
Is this ever used? Oh, if there's a table as a value inside an array? Could you add a test for that? Speaking of that, it would be good to also have a test for regular arrays (of strings or ints, for example).

Reply:
Sure, will do.